README.md

```diff
@@ -1,18 +1,27 @@
 # census
 
-Serverless deployment steps:
-<ul>  
+Type in :  `npm install -g serverless`  if you are running Serverless for the first time.
+ 
+ For basic Serverless deployment:
+ 
+1.Run CMD from local directory
 
-<li>Adjust implementation logic in handler.js
-<li>Adjust serverless.yml accordingly with function name
+2.Adjust implementation logic in handler.js
 
-</ul>
-Type the following into the command line:
-<ol>
-<li>serverless config credentials --provider aws --key *YOUR KEY* --secret *YOUR SECRET KEY* --overwrite</bold>
-<li>serverless deploy
+3.Adjust serverless.yml accordingly with function specifications
+
+4.Type the following into the command line:
+* `serverless config credentials --provider aws --key *YOUR KEY* --secret *YOUR SECRET KEY* --overwrite`
+* `serverless deploy`
+
+For **Serverless-Offline**:
+
+1. Type in: `sls offline` (Plug-in must be configured into serverless.yml file)
```

Comment:
is there onfiguration that still needs to be added for the serverless-offline plugin?

Reply:
The only necessary adjustment is in the _serverless.yml_ file, which I updated accordingly (under plugins).